ckanext/xloader/utils.py

```diff
@@ -2,6 +2,7 @@
 
 import json
 import datetime
+from rq import get_current_job
```

Comment:
you added get_current_job, but i don't see any reference in this file, is it meant to be part of anther enhancement?

Reply:
@duttonw my bad, it was for a really bad implementation of direct chaining with the validation job and xloader job. Before I found the `at_front` from the RQ lib. import has been removed.